### DISCUSSION TITLE
milvus2.1.4集群部署如何调优

### DISCUSSION THREAD

**Discussion by wlj521** [milvus-io/milvus#20255] [Ideas & Feature requests]
k8s集群部署的方式
向量5百万
索引FLAT
维度512
queryNode：16C32G * 2
调参：
memoryLimit = 10G
cpuRatio = 80
压测效果
QPS：65，rt99 517ms
求教大佬有没有调优方案

**Comment by yhmo** [comment]
调优这个事情一言难尽。
- FLAT索引其实就是没索引，纯暴搜超慢，除非是要求100%召回率才用。如果不是非要100%召回，可以考虑HNSW。
- 根据这个工具 https://milvus.io/tools/sizing ， 参考一下所需的资源再做部署
- 如果数据量是固定的，调用search时consistency_level不要设成strong
